doc/source/reference/c-api.array.rst

```diff
@@ -3432,13 +3468,25 @@ Miscellaneous Macros
 
     Returns the reference count of any Python object.
 
-.. c:function:: PyArray_XDECREF_ERR(PyObject \*obj)
+.. c:function:: PyArray_DiscardWritebackIfCopy(PyObject* obj)
+
+    Reset the :c:data:`NPY_ARRAY_WRITEBACKIFCOPY` and deprecated
+    :c:data:`NPY_ARRAY_UPDATEIFCOPY` flag. Also resets the
+    :c:data:`NPY_ARRAY_WRITEABLE` flag on the base object. This is
+    useful for recovering from an error condition when
+    writeback semantics are used, but will lead to wrong results.
```

Comment:
Wording isn't great here - I read this as "This is useful ..., but will lead to wrong results", which is just confusing

Reply:
how about "useful for recovering from an exception when writeback semantics are used. Pending changes to the base array will be thrown away"